patterns/roks/variables.tf

```diff
@@ -281,6 +282,30 @@ variable "operating_system" {
   }
 }
 
+variable "use_private_endpoint" {
+  type        = bool
+  description = "Set this to true to force all cluster related api calls to use the IBM Cloud private endpoints."
+  default     = false
+}
+
+variable "verify_worker_network_readiness" {
```

Comment:
In SLZ this has to be "false" since it is a private-only cluster, there is no use of having this as a variable, just hard-code in config.tf.